source/value-object.js

```diff
@@ -5,13 +5,12 @@ Space.messaging.Serializable.extend(Space.domain, 'ValueObject', {
   },
 
   _hasSameValues: function(other) {
-    var hasSameValues = true;
-    for(var key in this.fields()) {
-      if(this[key] instanceof Space.domain.ValueObject) {
-        if(!this[key].equals(other[key])) hasSameValues = false;
-      }
-      else {
-        if(this[key] !== other[key]) hasSameValues = false;
+    let hasSameValues = true;
```

Comment:
Seems like it would be better to start this off as false since it's unknown at this point